persistence/sql/persister_recovery.go

```diff
@@ -6,16 +6,13 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/ory/kratos/corp"
-	"github.com/ory/kratos/identity"
-
 	"github.com/gobuffalo/pop/v5"
 	"github.com/gofrs/uuid"
-
-	"github.com/ory/x/sqlcon"
-
+	"github.com/ory/kratos/corp"
```

Comment:
```suggestion

	"github.com/ory/kratos/corp"
```

Because we use `goimports -local github.com/ory` instead of only `goimports`. The local flag results in "local" imports to be grouped on the bottom of the imports, so you have three sections of imports: stdlib, third party, and first party.

Reply:
To run it locally, just use `make format`.